src/test/java/com/microsoft/sqlserver/jdbc/TestResource.java

```diff
@@ -43,6 +43,7 @@ protected Object[][] getContents() {
             {"R_lengthTruncated", " The inserted length is truncated or not correct!"},
             {"R_timeValueTruncated", " The time value is truncated or not correct!"},
             {"R_invalidErrorMessage", "Invalid Error Message: "},
+            {"R_kerberosNativeGSSFailure", "No valid credentials provided (Mechanism level: Failed to find any Kerberos tgt)"},
```

Comment:
Is `tgt` an abbreviation for target? If so, can this be spelled out fully?

Reply:
No, `tgt` means `ticket granting ticket`. This is an error string thrown by the GSS API, which we look for. Changing this string will cause the test to fail.